fs/config/config.go

```diff
@@ -274,6 +267,61 @@ func loadConfigFile() (*goconfig.ConfigFile, error) {
 		return goconfig.LoadFromReader(bytes.NewBuffer(b))
 	}
 
+	if len(configKey) == 0 {
+		pwc := fs.Config.PasswordCommand
+
+		if pwc == "" {
```

Comment:
We shouldn't need to read the environment variable here - `flags.StringVarP` should have loaded it from the env var already?

Ah, I see the problem, the variable associated with `--password-command` is `RCLONE_PASSWORD_COMMAND` not `RCLONE_CONFIG_PASS_COMMAND`.

So really we should be using `RCLONE_PASSWORD_COMMAND` everywhere.

The tests can then set `fs.Config.PasswordCommand` directly.